src/battle_script_commands.c

```diff
@@ -15765,7 +15765,7 @@ bool32 IsMoveAffectedByParentalBond(u32 move, u32 battler)
 {
     if (move != MOVE_NONE && move != MOVE_STRUGGLE
         && gBattleMoves[move].split != SPLIT_STATUS
-        && !gBattleMoves[move].strikeCount > 2)
```

Comment:
on upcoming this is `gBattleMoves[move].strikeCount <= 2` from 3412 (iirc). Which is correct?

Reply:
It shouldn't, because it Parental Bond doesn't affect Multi-hit moves, and the `<= 2` means that it would affect moves that hit twice.

Reply:
might be worth fixing on upcoming too, then

Reply:
Will do once I merge master into upcoming.